gcloud/connection.py

```diff
@@ -391,7 +391,8 @@ def api_request(self, method, path, query_params=None,
             target_object=_target_object)
 
         if not 200 <= response.status < 300:
-            raise make_exception(response, content)
+            raise make_exception(response, content,
+                                 request_string=method+' '+url)
```

Comment:
I'm surprised the linter didn't complain about the lack of spaces around the `+` signs.